ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -241,7 +241,7 @@ def test_missing_startup(self, expire_cert_critical):
             assert (err_msg1 in result.stderr_text
                     and err_msg2 in result.stderr_text)
         else:
-            assert warn_msg in result.stdout_text
+            assert warn_msg in result.stderr_text
```

Comment:
The exact message is
`WARNING: No selftests configured in /var/lib/pki/pki-tomcat/ca/conf/CS.cfg (selftests.container.order.startup).` but we are looking for
`WARNING: No selftests configured in /var/lib/pki/pki-tomcat/conf/ca/CS.cfg (selftests.container.order.startup)`.
This is equivalent since both are symlinks to the same file:
```
[root@server ~]# readlink -f /var/lib/pki/pki-tomcat/conf/ca/CS.cfg
/etc/pki/pki-tomcat/ca/CS.cfg
[root@server ~]# readlink -f /var/lib/pki/pki-tomcat/ca/conf/CS.cfg
/etc/pki/pki-tomcat/ca/CS.cfg
```
I suggest to shorten warn_msg and focus only on the beginning of the expected message:
```
warn_msg = "WARNING: No selftests configured in "
```